docs/docs/webchat/webchat-covercomponent.md

```diff
@@ -0,0 +1,148 @@
+---
+id: webchat-covercomponent
+title: Cover Component
+---
+
+## 
+
+The Cover component enables the user to authenticate when starting a conversation. In that way, you can retrieve relevant information at the very beginning of the interaction, such as the user's name or e-mail address.
+
+<details>
+<summary>Example</summary>
```

Comment:
I think it's not necessary to have two screenshots of the coverComponent, also they look very similar.

Reply:
OK